examples/llama-3/instruct-lora-8b.yml

```diff
@@ -0,0 +1,76 @@
+base_model: meta-llama/Meta-Llama-3-8B-Instruct
+model_type: LlamaForCausalLM
+tokenizer_type: AutoTokenizer
+
+load_in_8bit: true
+load_in_4bit: false
+strict: false
+
+chat_template: llama3
+datasets:
+  - path: fozziethebeat/alpaca_messages_2k_test
+    type: chat_template
+    chat_template: llama3
+    field_messages: messages
+    message_field_role: role
+    message_field_content: content
```

Comment:
Do we really need to assign the chat_template twice in dataset and outside? I'm testing this PR. Any difference between the two "chat_template" settings?

Reply:
we already have some handling when using this with sharegpt and chatml, so I've updated that to handle it automatically for the general case here: https://github.com/OpenAccess-AI-Collective/axolotl/pull/1664